src/content/doc-surrealdb/index.mdx

```diff
@@ -35,8 +47,105 @@ import DarkGettingStarted from "@img/image/dark/gettingstarted.png";
+
+We've included [demo dataset](/docs/surrealql/demo) which you can download and [<code>import</code> ](/docs/surrealdb/cli/import)into your SurrealDB instance.
+
+
+
+SurrealDB supports all [realeased versions](/releases) from `1.0.0` onwards. <span>The current stable version of SurrealDB </span><span> is <a href='/releases'> <Version /> </a></span>
+
+
+<div class="border-b border-border/40 my-8"></div>
+
+<div class="flex gap-4 items-center justify-center">
+    <a href="/docs/surrealdb/installation" class=" border border-border/40 flex-1 inline-flex items-center justify-center px-6 py-3 text-base font-medium bg-[#ff00A0] hover:bg-[#ff00A0]/90 rounded-lg transition-colors">
+        Install SurrealDB <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="inline-block ml-1"><path d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"></path><polyline points="7 10 12 15 17 10"></polyline><line x1="12" y1="15" x2="12" y2="3"></line></svg>
+    </a>
+    <a href="/docs/surrealdb/introduction/start" class="flex-1 inline-flex items-center justify-center text-base font-medium text-text hover:text-text/80 transition-colors">
+        Get Started with SurrealDB →
+    </a>
+</div>
+
+
+<div class="border-b border-border/40 my-8"></div>
+
+
+<h2 class="text-4xl mt-10 font-thin">
+    Getting Started
+</h2>
+
+<Boxes wider class="pt-2">
+                    <IconBox
+                        title="SurrealDB University"
+                        description="Dive into our collection of recorded videos, live streams, and expert talks. Enhance your learning experience by accessing a variety of video content that covers essential SurrealDB topics and industry insights."
+                        href="/learn"
+                        icon={{ light: LightUniversity, dark: DarkUniversity }}
+                    />
+                    <IconBox
+                        title="The SurrealDB Book"
+                        description="Embark on a journey through the ins and outs of SurrealDB as you follow an immersive story crafted to make learning a new database feel as enjoyable as sitting back and watching your favourite movie."
+                        href="/learn/book"
+                        icon={{ light: LightBook, dark: DarkBook }}
+                    />
+                    <IconBox
+                    title="Surreal Sidekick"
+                    description="Surreal Sidekick is your personal assistant that helps you learn SurrealDB by providing you with a set of questions and answers that cover SurrealDB, SurrealQL and all the other products we offer."
```

Comment:
I think "a set of" makes it sound like Sidekick's answers are set in stone, how about this: "by generating answers to your questions covering SurrealDB, SurrealQL and all the other products we offer."